superset/models/core.py

```diff
@@ -888,7 +889,7 @@ def get_schema_access_for_file_upload(  # pylint: disable=invalid-name
     def sqlalchemy_uri_decrypted(self) -> str:
         try:
             conn = make_url_safe(self.sqlalchemy_uri)
-        except (ArgumentError, ValueError):
+        except DatabaseInvalidError:
```

Comment:
I almost asked about why the old exceptions here aren't valid any more. To answer that question for other potential reviewers: we're already reraising all exceptions to `DatabaseInvalidError` here: https://github.com/apache/superset/blob/44557f5a23d5a6b8f7d6cc267f0d43337c36cd76/superset/databases/utils.py#L120-L121